articles/contributing-style-guide.md

```diff
@@ -234,15 +232,15 @@ We suggest:
 
 ### Private or internal names ###
 
-In many cases, a name is intended strictly for use internal to a library or project, and is not a guaranteed part of the API offered by a library.
+In many cases, a name is intended strictly for internal use in a library or project. It is not a guaranteed part of the API that a library offers.
 It is helpful to clearly indicate that this is the case when naming functions and operations so that accidental dependencies on internal-only code are made obvious.
-If an operation or function is not intended for direct use, but rather should be used by a matching callable which acts by partial application, consider using a name starting with the `internal` keyword for the callable that is partially applied.
+There are operations and functions that aren't intended for direct use. They can be used by a matching callable which acts by partial application. Then consider using a name that starts with the `internal` keyword for the callable that is partially applied.
 
 # [Guidance](#tab/guidance)
 
-We suggest:
+We recommend:
 
-- When a function, operation, or user-defined type is not a part of the public API for a Q# library or program, ensure that it is marked as internal by placing the `internal` keyword before the `function`, `operation`, or `newtype` declaration.
+- When functions, operations, or user-defined types are not part of the public API for a Q# library or program, mark it as internal by placing the `internal` keyword before the `function`, `operation`, or `newtype` declaration.
```

Comment:
Subject–verb agreement? Should "mark it" be "mark them"?

Reply:
Correct, I have made the change.